webui/react/src/pages/Cluster/ClusterOverallStats.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react';
+
+import Grid, { GridMode } from 'components/Grid';
+import OverviewStats from 'components/OverviewStats';
+import Section from 'components/Section';
+import { useStore } from 'contexts/Store';
+import { ShirtSize } from 'themes';
+import { ResourceType } from 'types';
+
+export const ClusterOverallStats: React.FC = () => {
+  const { agents, cluster: overview, resourcePools } = useStore();
+
+  const auxContainers = useMemo(() => {
+    const tally = {
+      running: 0,
+      total: 0,
+    };
+    resourcePools.forEach(rp => {
+      tally.total += rp.auxContainerCapacity;
+      tally.running += rp.auxContainersRunning;
+    });
+    return tally;
+  }, [ resourcePools ]);
+
+  const [ cudaTotalSlots, rocmTotalSlots ] = useMemo(() => {
+    return resourcePools.reduce((acc, pool) => {
+      let index;
+      switch (pool.slotType) {
```

Comment:
per https://determined-ai.slack.com/archives/CLCE8D998/p1651701445031029
I believe we should treat CPU slots same as CUDA/ROCM here 
cc @hamidzr

Reply:
the change I'm looking at here might be outdated at this point but maybe bringing [this change](https://github.com/determined-ai/determined/pull/4080/files#diff-36cdee0a95644a8d46011b3c54a395f2e849b4b147817bfee1d0cb816c70ffd8R98) in would do this?

Reply:
I already updated how we calculate total slots according to Hamid's change, feel free to verify and bring up any questions or concerns

Reply:
I already updated how we calculate total slots according to Hamid's change, feel free to verify and bring up any questions or concerns